### PR TITLE
[Shiori] Add Shiori install guide

### PR DESCRIPTION
**Pull request by Systemsemaphore**
Hello all,
I need help with this guide before #837 can be closed.

I've written guide_shiori.rst (my first guide) based on guide_gotify.rst. I have successfully tested

- [x] installation

- [x] configuration (setting up a daemon)

- [x] accessing the web-ui and logging in

- [x] creating a new account

- [x] creating a bookmark

- [x] ~~However, I didn't manage to get it up and running behind a sub path~~ Sub path doesn't work yet, see go-shiori/shiori#39

It's working with [https://shiori.uber.space](https://shiori.uber.space) and `uberspace web backend set / --http --port 8080` right now. ~~I've tried `uberspace web backend set /shiori/ --http --port 8080` with and without `--remove-prefix` before, both fail with error 404. Help would be appreciated.~~

### CODE REVIEW

**Comment by Systemsemaphore** [comment]
go-shiori/shiori#39

Awkward. I'll go fix _Prerequisites_

bart

**Comment by luto** [comment]
Thank you! I'll take a look this week :)

File: source/guide_shiori.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+.. _webbackend: https://manual.uberspace.de/web-backends.html
+
+Installation
+============
+
+Like a lot of Go software, Shiori is distributed as a single binary. Download Shiori's latest `release <https://github.com/go-shiori/shiori/releases/latest>`_ and make sure that the file can be executed.
+
+::
+
+  [isabell@stardust ~]$ mkdir ~/shiori && cd ~/shiori
```
Comment (luto) [inline source/guide_shiori.rst:52]: The app writes all data to `~/.local/share/shiori` and there aren't any configuration files. What do you think about saving the binary as `~/bin/shiori` using `wget ... -O ~/bin/shiori` and skipping the directory step entirely?

File: source/guide_shiori.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+-----------
+
+.. note:: To use a different port instead of 8080, you need to add ``-p <portnumber>`` to the command in your ``shiori.ini`` file. Remember to adjust the port for the web backend as well.
+
+Create ``~/etc/services.d/shiori.ini`` with the following content.
+
+.. code-block:: ini
+
+  [program:shiori]
+  command=/home/isabell/shiori/shiori-linux-amd64-stretch serve
```
Comment (luto) [inline source/guide_shiori.rst:79]: if you move the binary to `~/bin/shiroi`, shorten this to
```suggestion
  command=shiori serve
```

**Review by luto — CHANGES_REQUESTED** [review]
Awesome! :tada: The guide works great :) I left a couple of suggestions to make it even more streamlined. Most of them can be applied by just pressing the commit button in the github web interface. Let me know what you think!

**Comment by Systemsemaphore** [comment]
Saving the binary as `~/bin/shiori` using `wget ... -O ~/bin/shiori` worked. Had to reconfigure the service as well but it's up and running again on my test account. I've added the suggested lines and looking at the preview right now I think there aren't any unresolved suggestions left. Please have another look, thanks in advance. In theory, it should also be possible to setup a backend for a specific domain and use the Document Root for something else besides Shiori but 1. [it's not encouraged](https://blog.uberspace.de/die-sache-mit-den-virtuellen-documentroots/) and 2. [it's against the rules for guides](https://github.com/Uberspace/lab/blob/master/STYLE.md) so I guess this will be an experiment just for me :)

**Comment by luto** [comment]
Thank youu! :tada:

**Comment by luto** [comment]
I can't find an email address on your github profile. Please ping hallo@uberspace.de about your reward :)
